WinImageBuilder.psm1

```diff
@@ -504,6 +504,31 @@ function Download-CloudbaseInit {
     }
 }
 
+function Download-ZapFree {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [string]$resourcesDir,
+        [Parameter(Mandatory=$true)]
+        [string]$osArch
+    )
+    $ZapFreePath = "$resourcesDir\zapfree.exe"
+    $ZapFree32Path = "$resourcesDir\zapfree32.exe"
+    $ZapFreeZipPath = "$resourcesDir\ntfszapfree.zip"
+    Write-Log "Downloading ntfszapfree..."
+

```

Comment:
@felfert you are required to add here a secure tls connection setting
[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12

Otherwise, the download will fail with this error:

```
6/14/2019 1:30:46 PM - Downloading ntfszapfree...
WARNING: Exception calling "DownloadFile" with "2" argument(s): "The request was aborted: Could not create SSL/TLS secure channel."
WARNING: Exception calling "DownloadFile" with "2" argument(s): "The request was aborted: Could not create SSL/TLS secure channel."
WARNING: Exception calling "DownloadFile" with "2" argument(s): "The request was aborted: Could not create SSL/TLS secure channel."
6/14/2019 1:30:59 PM - Exception calling "DownloadFile" with "2" argument(s): "The request was aborted: Could not create SSL/TLS secure channel."
```

Reply:
Hmm strange. Why does Download-CloudbaesInit not require this as well?
Perhaps your download server still offers Tls1?
Anyway: done

Reply:
I have added a separate PR for the TLS enforcement: https://github.com/cloudbase/windows-openstack-imaging-tools/pull/273/files that enables Tls1.2 globally.

GitHub now requires TLS1.2, while cloudbase.it or fedora virtio repo do not.

Reply:
Ok, so I reverted my last commit